### PR TITLE
ability to provide locale to MapFilter element

### LINKED ISSUE #72: Explicit Language Selection
Feature Request:

Allow MapFilter to take a property `locale` or similar that can be used to explicitly specify the language used for translation, as opposed to solely getting this from the browser.

### PR DESCRIPTION
**Pull request by okdistribute**
fixes #72

### CODE REVIEW

File: src/containers/MapFilter.js [hunk 1]
```diff
@@ -192,6 +191,10 @@ class MapFilter extends React.Component {
     super(props)
     const stateOverride = pick(props, controllableProps)
     const controlledStoreEnhancer = controlledStore(this.handleChange, stateOverride)
+    var lang = this.locale
```
Comment (GoGoCarl) [inline src/containers/MapFilter.js]: Should this be `this.props.locale`?

**Review by GoGoCarl — COMMENTED** [review]
See one question below about the constructor, but otherwise this is exactly what I was expecting, thanks!

File: src/containers/MapFilter.js (hunk 1 above)
Comment (okdistribute) [reply src/containers/MapFilter.js]: ah good catch, probably!

**Comment by okdistribute** [comment]
ok @GoGoCarl going to fix that and merge then!

**Pull request merged**